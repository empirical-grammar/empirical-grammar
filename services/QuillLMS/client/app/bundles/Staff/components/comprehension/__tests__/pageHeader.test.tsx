import * as React from 'react';
import { shallow } from 'enzyme';

import PageHeader from '../shared/pageHeader';

describe('PageHeader component', () => {
  const mockProps = {
    header: '',
    title: '',
    notes: ''
  }
  const container = shallow(<PageHeader {...mockProps} />);

  it('should render PageHeader', () => {
    expect(container).toMatchSnapshot();
  });
});
