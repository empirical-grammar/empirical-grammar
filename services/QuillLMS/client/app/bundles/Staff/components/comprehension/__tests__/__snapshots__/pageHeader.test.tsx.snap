// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader component should render PageHeader 1`] = `
<section
  className="comprehension-page-header-container"
>
  <h2 />
  <h3 />
  <h4 />
</section>
`;
